### ISSUE TITLE
How to configure classification labels (based on return ranking) and adapt model training for classification tasks

### ISSUE THREAD

**Issue by scnu001** [microsoft/qlib#2137]
## ❓ Questions and Help
How can I configure the label in the task config to generate discrete classification labels instead of continuous regression values? Specifically, I want to discretize the returns based on their cross-sectional ranking.
Once the labels are discrete, how should I configure the model and loss function to perform classification training?
And How should the backtesting module be adapted for classification outputs?

**Comment by scnu001** [comment]
I find we can add binary in model loss like this '''model_config = {
    "class": "LGBModel",
    "module_path": "qlib.contrib.model.gbdt",
    "kwargs":  {
        "loss": "binary",
        "colsample_bytree": 0.8879,
        "learning_rate": 0.0421,
        "subsample": 0.8789,
        "lambda_l1": 205.6999,
        "lambda_l2": 580.9768,
        "max_depth": 8,
        "num_leaves": 210,
        "num_threads": 20,
        "seed": 688
}
}''' but when i check the dataset like this code  handler = dataset.handler
print(handler.get_cols())                     
print(handler.get_cols('feature'))            
print(handler.get_cols('label'))              
print(df_train['LABEL0'].head()) i find that there are only label0 yield so how can model classify train. Does model use yield if >0 or not to class?
